backend/yugioh/utils.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+import requests
+from django.conf import settings
+
+
+def fetch_and_save_image(image_url):
+
+    external_image_url = image_url
+
+    local_image_name = image_url.split('/')[-1]
+
+    media_path = os.path.join(settings.MEDIA_ROOT, 'yugioh_card_images')
+
+    image_file_name = os.path.join(media_path, local_image_name)
+
+    headers = {"User-Agent": "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0",
```

Comment:
Are you sure you need the headers?

Reply:
> Are you sure you need the headers? What is their purpose in this case? Also, if yes, move them inside the if (before the request).

i was not sure, it actually works without them, and reformated the code.

Thanks.